.circleci/config.yml

```diff
@@ -124,31 +124,28 @@ jobs:
           paths:
             - app/dist
 
-  # Build Gaia from the SDK source code specified by tasks/build/Gaia/COMMIT.sh.
+  # Build Gaia from the SDK source code specified by tasks/build/Gaia/VERSION
   buildGaia:
     executor: go
     steps:
       - checkout
       - restore_cache:
-          key: v8-Gaia-{{ checksum "tasks/build/Gaia/COMMIT.sh" }}
-
-      # If Gaia isn't in the cache then build it.
+          key: v8-gaia-{{ checksum "tasks/build/Gaia/VERSION" }}
       - run: |
           if [ ! -d /tmp/gaia ]; then
             cd tasks/build/Gaia
-            . ./COMMIT.sh
-            TARGET=/tmp/gaia PLATFORM=linux ./build.sh
+            TAG=`cat ./VERSION` TARGET=/tmp/gaia PLATFORM=linux ./build.sh
```

Comment:
why linux only?

Reply:
It runs on top of alpine so it's the only supported OS

Reply:
is that documented somewhere ?

Reply:
It's maybe not that straightforward now that you ask.
we are using the official `circleci/golang:1.11` 
that is built on top of `golang:1.11.1` 
that is built on top of `alpine:stable`